model_definition.md

```diff
@@ -204,7 +204,15 @@ Genome concentration measurements can vary between sites, and even within a site
 
 $$\log[c_{ijt}] \sim \mathrm{Normal}(\log[M_{ij} C_i(t)], \sigma_{cij})$$
 
-Both $M_{ij}$ and $\sigma_{cij}$ are modeled as site-level random effects.
+Both $M_{ij}$ and $\sigma_{cij}$ are estimated hierarchically such that:
+
+$$\log(M_{ij}) \sim \mathrm{Normal}(0, \sigma_m)$$
+
+such that $M_{ij}$ is centered around 1 and
+
+$$\log(\sigma_{cij}) \sim \mathrm{Normal}(\log(mode_{\sigma}), \sigma_{ c, \log})$$
```

Comment:
Can we notate the mode and sd for $\log(\sigma\_{cij)}$ differently? Maybe something like $\hat{\sigma}\_c$ for the mode and $\sigma\_{\log \sigma\_c}$ for the sd?